packages/code-du-travail-frontend/src/sources.js

```diff
@@ -1,24 +0,0 @@
-// mapping elastic search source type -> route name
```

Comment:
Why did you remove it ?

Reply:
argh PR splitting...

Reply:
I see that in the PR "Unify Content Route" ;)

Reply:
ok je remix :)